### DISCUSSION TITLE
Subversion gateway returns E900003:  SVN is being set up for this repository

### DISCUSSION THREAD

**Discussion by theultramage** [community/community#28729] [Other Feature Feedback, Questions, & Ideas]
I use the svn interface to https://github.com/freebsd/freebsd-ports to minimize the storage footprint and storage activity on my mini server when doing updates. Since their migration to github in 2021, everytime I tried accessing it, it would malfunction several times with `E175002: Unexpected server error 500 'Internal Server Error'` (presumably a serverside execution cutoff) before finally returning the answer. I've gotten used to it.

Last time I updated from this repo was several weeks ago (2022-07-25). When I tried it again now, strangely, I instead got several `504 'Gateway Timeout'` replies, before finally settling on `E900003: SVN is being set up for this repository. Check back soon!`. I have been receiving this reply for the past 4 days so far. This error code and error message give no results on web search.

The repository has 2m changes and they're all pretty small. It should not be taking this long, even if the whole repo is being resynced for some reason. Repeatedly invoking remote 'svn info' eventually succeeds and returns 'Last Changed Date: 2022-08-10', so the metadata is there and _is_ up to date. I tried `svn diff -c 500' on the remote base url to see if at least some of the data is there. First time around, I actually got the diff as the reply. Subsequent reruns of the command gave me the abovementioned error. This makes me think that whatever CDN github uses is broken.

**Comment by sskras** [comment]
Just received the same response:
```
$ date; svn export https://github.com/freebsd/freebsd-src/trunk/lib/libfetch
24 Oct 2022 13:10:39
svn: E900003: SVN is being set up for this repository. Check back soon!
```

**Comment by theultramage** [comment]
The repository has 5.6m items and its size is around 2GB.
Github support did get back to me in a month:
> It looks as though this repository may be large enough that it is stretching the limits of the SVNBridge tool. We're looking into it, but it's unlikely there will be a short term fix. We've triggered an update of the mapping on our end, and we hope that will improve the experience for you. If it doesn't, we would have to recommend using Git at this time.

Whatever the 'mapping update' was, it allowed me to perform 2 small updates, then it fell apart again.
There has been no improvement since then, and my support ticket is now archived.

**Reply by sskras** [reply]
Thanks for the update!

**Reply by snomos** [reply]
Just got the same error, thanks for doing the work of finding an answer to it.
